File: Java/1009 - Salário com Bônus.java
import java.io.IOException;
import java.util.*;

public class Main{
    public static void main(String[] args){
        String nome;
        double salarioFixo,vendas, salarioBonus;
        
        Scanner valor = new Scanner (System.in);
        nome = valor.next();
        salarioFixo = valor.nextDouble();
        vendas = valor.nextDouble();
        
        salarioBonus = (vendas * 0.15) + salarioFixo;
        
        System.out.println(String.format("TOTAL = R$ %.2f", salarioBonus));
        
    }
}

File: C#/1009 - Salário com Bônus.cs
using System;

namespace SalarioBonus{
    class Program{
        static void Main(string[] args){
            string nomeVend;
            double salarioFixo, totalVendas, salarioBonus;
            
            nomeVend = Convert.ToString(Console.ReadLine());
            salarioFixo = Convert.ToDouble(Console.ReadLine());
            totalVendas = Convert.ToDouble(Console.ReadLine());
            
            salarioBonus = (totalVendas * 0.15) + salarioFixo;
            
            Console.WriteLine(string.Format($"TOTAL = R$ {salarioBonus.ToString("0.00")}"));
        }
    }
}
